packages/slice-machine/server/src/api/tracking.ts

```diff
@@ -45,11 +45,11 @@ export function sendEvents(
 
 export default async function handler(req: RequestWithEnv): Promise<void> {
   const data = req.body as TrackingEvents;
-  if (isTrackingEvent(data)) {
-    const analytics = new Analytics(
-      process.env.NEXT_PUBLIC_SM_UI_SEGMENT_KEY ||
-        "Ng5oKJHCGpSWplZ9ymB7Pu7rm0sTDeiG"
-    );
+  const trackingEnabled =
```

Comment:
Would that work ?
```suggestion
  const trackingEnabled = typeof req.env.manifest.tracking == 'boolean' ? req.env.manifest.tracking : false
```

Reply:
if it's undefined (default), then it would be false, we could change that to true :)

Reply:
ah right 😂

Reply:
I guess it's not that important.